hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/telemetry/TraceExtractingMessageInterceptor.kt

```diff
@@ -50,13 +47,13 @@ class TraceExtractingMessageInterceptor(private val objectMapper: ObjectMapper)
       attributes.extractTelemetryContextFromValues().startSpan(spanName)
     }
 
-  private fun startSpanFromAttributesInPayload(payload: String?): Span {
+  private fun startSpanFromAttributesInPayload(payload: String?): Span? {
     val attributes = objectMapper.readValue(
       objectMapper.readTree(payload).at("/MessageAttributes").traverse(),
       object : TypeReference<MutableMap<String, MessageAttribute>>() {},
     )
-    val spanName = attributes["eventType"]?.let { "RECEIVE ${it.Value}" } ?: "RECEIVE"
-    return attributes.extractTelemetryContext().startSpan(spanName)
+    val spanName = attributes?.get("eventType")?.let { "RECEIVE ${it.Value}" } ?: "RECEIVE"
```

Comment:
this is the bugfix - `attributes` was coming through as `null` and thus then throwing a null pointer exception